### PR TITLE
kernel: init at 4.4

### PR DESCRIPTION
**Pull request by NeQuissimus**
Initialize the kernel 4.4 line

### CODE REVIEW

**Comment by mogorman** [comment]
fails because of time limit probably good

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -10047,6 +10047,15 @@ let
       ];
   };
 
+  linux_4_4 = callPackage ../os-specific/linux/kernel/linux-4.4.nix {
+    kernelPatches = [ kernelPatches.bridge_stp_helper ]
+      ++ lib.optionals ((platform.kernelArch or null) == "mips")
+      [ kernelPatches.mips_fpureg_emu
```
Comment (aristidb) [inline pkgs/top-level/all-packages.nix]: I suppose we don't actually know if these patches still apply ;)

Comment (NeQuissimus) [inline pkgs/top-level/all-packages.nix]: I do not :-)  they seem to have been copied from kernel to kernel, so I kept with the tradition ;-)

**Comment by NeQuissimus** [comment]
Don't merge this yet

`./lib/modules/4.4/build` cannot be unlinked, I will take a look unless somebody knows why.

**Comment by dezgeg** [comment]
`modDirVersion` needs to be set to `4.4.0`.

**Comment by NeQuissimus** [comment]
Correct, it works now, I have it installed.

**Comment by NeQuissimus** [comment]
@shlevy seems to have done this in 44274f62f59a7f0b7460b098c1eb50b54f449565, closing PR.

**Comment by shlevy** [comment]
Oops, sorry!

**Comment by NeQuissimus** [comment]
No worries, just cleaning up :)

**Pull request closed without merging**